packages/ui/src/core/robot/robot.component.tsx

```diff
@@ -36,7 +33,7 @@ const Robot = intrinsicComponent<RobotProps, HTMLSpanElement>(
 );
 
 export const defaultProps = {
-  status: Status.Happy,
+  status: Status.Success,
 };
 
 Robot.defaultProps = defaultProps;
```

Comment:
We need to change this Robot component to another name because we aren't using robots now. Also we need to update it's story